fe/fe-core/src/main/java/com/starrocks/alter/LakeTableSchemaChangeJob.java

```diff
@@ -103,6 +103,9 @@ public class LakeTableSchemaChangeJob extends AlterJobV2 {
     // shadow index id -> shadow index short key count
     @SerializedName(value = "indexShortKeyMap")
     private Map<Long, Short> indexShortKeyMap = Maps.newHashMap();
+    // partition id -> shadow shard group id
+    @SerializedName(value = "shardGroupIdMap")
+    private Map<Long, Long> shardGroupIdMap = Maps.newHashMap();
```

Comment:
Schema change jobs created before the upgrade didn't have this map, will that be a problem?

Reply:
i also discovered this, it's fixed by checking this field